frontend-admin/src/pages/routes/routes-page.tsx

```diff
@@ -1,15 +1,33 @@
 import React from 'react';
+import { useQuery } from 'react-query';
 import Card from '../../components/card/card';
 import './routes-page.scss';
 
+interface Route {
+  id: number;
+  name: string;
+}
+
+const fetchRoutes = async () => {
+  const response = await fetch('http://localhost:8000/routes');
```

Comment:
Eventually, we will want to dynamically switch between this dev server and a prod server. See the app for an example.

Reply:
yep, just hardcoding for demo